src/openms/source/ANALYSIS/XLMS/OpenPepXLLFAlgorithm.cpp

```diff
@@ -174,6 +178,8 @@ using namespace OpenMS;
 
     number_top_hits_ = static_cast<Int>(param_.getValue("algorithm:number_top_hits"));
     deisotope_mode_ = static_cast<String>(param_.getValue("algorithm:deisotope"));
+    use_sequence_tags_ = (param_.getValue("algorithm:use_sequence_tags") == "true" ? true : false);
```

Comment:
`use_sequence_tags_ = (param_.getValue("algorithm:use_sequence_tags") == "true")` is enough